app/src/androidTest/java/net/skyscanner/backpack/docs/DocsRegistry.kt

```diff
@@ -115,8 +116,9 @@ object DocsRegistry {
     ComposeScreenshot("Text - Compose - Hero", "hero"),
     ComposeScreenshot("Text - Compose - Heading", "heading"),
     ComposeScreenshot("Text - Compose - Body", "body"),
-    ViewScreenshot("Text Field - Default", "default"),
-    ViewScreenshot("Text Field - With labels", "labels"),
+    ViewScreenshot("Text Field - View - Default", "default"),
+    ViewScreenshot("Text Field - View - With labels", "labels"),
+    ComposeScreenshot("Text Field - Compose", "default"),
```

Comment:
same here

Reply:
I'm not sure we actually benefit from having all these screenshots on the website.

Reply:
both designers & engineers use the website to see what the components support in code - the screenshots are particularly useful for design. a lot of the functionality isn't shown on the screenshots, and while you can see that there is some support for error, validation etc you can't see what it looks like/what's actually supported

Reply:
Added it!